unity/Assets/ZestySDK/Scripts/Internal/Utils/Constants.cs

```diff
@@ -7,5 +7,6 @@ public static class Constants {
         public const string CDN_URL = "https://cdn.zesty.xyz";
         public const bool PREBID = true;
         public const int MAX_PREBID_RETRIES = 4;
+        public const int PREBID_REFRESH_INTERVAL = 15; // 15 seconds
```

Comment:
Did we want to lower this to 10 to keep it in line with other SDKs? (unless there's a specific reason to be otherwise)